static/js/narrow.js

```diff
@@ -3,6 +3,7 @@ var narrow = (function () {
 var exports = {};
 
 var unnarrow_times;
+var is_in_private_narrow_now;
```

Comment:
Shouldn't this also change `narrow.deactivated`?  I'd also move it to the `narrow_state` module...

Reply:
@timabbott Ah, that makes sense. It turns out `narrow_state` actually already has a function for this (`narrowed_to_pms`), so adding a new one is unnecessary. I removed that commit and updated the other references to the new function.